_episodes_rmd/04-data-structures-part1.Rmd

```diff
@@ -376,6 +376,26 @@ These make sense. But what about
 str(cats$coat)
 ```
 
+> ## Tip: Renaming dataframe columns
+>
+> Dataframes have column names we can access them like we did with vectors
```

Comment:
Grammar: "Dataframes have column names we can access them like we did with vectors" -> "Data frames have column names, which can be accessed with the `names()` function."